03-GettingStarted/07-aitk/README.md

```diff
@@ -38,6 +38,9 @@ Great, now that we understand the flow, let's configure an AI agent to leverage
 
 ## Exercise: Consuming a server
 
+⚠️ **Note for macOS Users:**
+We're currently investigating an issue affecting dependency installation on macOS. As a result, macOS users won’t be able to complete this tutorial at this time. We’ll update the instructions as soon as a fix is available. Thank you for your patience and understanding!
```

Comment:
Hey @aprilgitten, I've seen a few other .md files in this repo use [GitHub's special blockquote syntax for alerts](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts).

```suggestion
> [!WARNING]
> Note for macOS Users. We're currently investigating an issue affecting dependency installation on macOS. As a result, macOS users won’t be able to complete this tutorial at this time. We’ll update the instructions as soon as a fix is available. Thank you for your patience and understanding!
```